v8/client/client.go

```diff
@@ -243,6 +243,45 @@ func (cl *Client) Destroy() {
 	cl.Log("client destroyed")
 }
 
+func (cl *Client) GetCCache() (*credentials.CCache, error) {
+	tgt, flags, skey, err := cl.sessionTGT(cl.Credentials.Realm())
+	if err != nil {
+		return nil, err
+	}
+	authTime, startTime, endTime, renewTime, _, err := cl.sessionTimes(cl.Credentials.Realm())
+	if err != nil {
+		return nil, err
```

Comment:
I'm not sure if an error needs to be wrapped here for additional context. Maybe not, but it can never hurt to add a little bit of information that this error happened while trying to get the ccache.

For example:

```go
return nil, fmt.Errorf("failed to get session times while getting cache (%w)", err)
```

Reply:
Fixed.